crossbuilder

```diff
@@ -386,6 +449,8 @@ create_container () {
     exec_container_root adduser $USERNAME sudo
     # set empty password for the user
     exec_container_root passwd --delete $USERNAME
+    # There is a chance, that some farch and build variables were not filled, due to missing packages on host machine. Use container to fill them if needed.
```

Comment:
This comment is repeated many times and is quite long; I suggest moving it at the beginning of the top of the `detect_farch_and_build_params_in_container_if_needed()` function only, and here just write "Ensure build variables are filled".

Please do the same below, where this comment is repeated again.